cms/test_utils/project/nonroot_urls.py

```diff
@@ -21,7 +21,11 @@
 
 urlpatterns += i18n_patterns(
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^content/', include('cms.urls')),
+    url(r'^content/', include(
+        'cms.urls',
+        app_name=get_cms_setting('APP_NAME'),
```

Comment:
Thanks @yakky, I will then push a fix for this in a separate branch along with other fixes.